src/Confluent.Kafka/Producer.cs

```diff
@@ -302,16 +303,17 @@ public Producer(IEnumerable<KeyValuePair<string, object>> config, bool manualPol
         ///     Initializes a new Producer instance.
         /// </summary>
         /// <param name="config">
-        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-        ///     topic configuration parameters should be specified via the "default.topic.config" config parameter.
+        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
+        ///     Topic configuration parameters should be specified via the "default.topic.config" config parameter.
         /// </param>
         public Producer(IEnumerable<KeyValuePair<string, object>> config)
             : this(config, false, false) {}
 
 
         /// <summary>
-        ///     Poll for callback events. Only use this method on producer
-        ///     instances where background polling has been disabled.
+        ///     Poll for callback events. You will not typically need
```

Comment:
Why was this changed?
Is there a case where you should call Poll() with background polling enabled?